tomviz/modules/Module.cxx

```diff
@@ -197,6 +212,13 @@ QJsonObject Module::serialize() const
     if (m_useDetachedColorMap) {
       json["colorOpacityMap"] = tomviz::serialize(d->detachedColorMap());
       json["gradientOpacityMap"] = tomviz::serialize(gradientOpacityMap());
+      QJsonObject boxJson;
+      auto transfer2DBox = d->detachedTransferFunction2DBox();
+      boxJson["x"] = transfer2DBox->GetX();
+      boxJson["y"] = transfer2DBox->GetY();
+      boxJson["width"] = transfer2DBox->GetWidth();
+      boxJson["height"] = transfer2DBox->GetHeight();
+      json["colorMap2DBox"] = boxJson;
```

Comment:
Seem like this is duplicated from DataSource?

Reply:
It didn't seem worth it to make separate functions to store/load 4 values.